assets/scripts/book.js

```diff
@@ -0,0 +1,161 @@
+/* eslint-disable func-names */
+/* eslint-disable camelcase */
+/* eslint-disable no-alert */
+/* eslint-disable array-callback-return */
+/* eslint-disable  consistent-return */
+
+
+const addnote = document.querySelector('.add_note');
+const formcontainer = document.querySelector('.form-container');
+
+const form = document.querySelector('form');
+const button = document.querySelector('button');
+let book_list = [];
+function Book(title, name, pages, readstatus = false) {
+  this.title = title;
+  this.name = name;
+  this.pages = pages;
+  this.readstatus = readstatus;
+}
+
+let read = '';
+
+Book.prototype.info = function () {
+  if (this.readstatus) {
+    read = 'Read!';
+    return read;
+  }
+  read = 'Not Read!';
+  return read;
+};
+Book.prototype.toggleStatus = function () {
+  this.readstatus = !this.readstatus;
+};
+
+
+if (localStorage.length < 1) {
+  book_list = [
+    {
+      title: 'Lord of the Rings', name: 'JRR Tolkien', pages: 300, readstatus: true,
+    },
+    {
+      title: 'Neuromante', name: 'William Gibson', pages: 200, readstatus: true,
+    },
+    { title: 'The Call of The Wild', name: 'Jack London', pages: '80' },
+    {
+      title: "Hitchiker's Guide to The Galaxy", name: 'Douglas Adams', pages: '250', readstatus: true,
+    },
+  ];
+} else {
+  const stored_list = localStorage.getItem('book_list');
+  book_list = JSON.parse(stored_list);
+}
+
+
+const render = function (template, node, container = document.createElement('div')) {
+  container.innerHTML = template;
+  node.appendChild(container);
+};
+
+
+book_list.forEach((book) => {
+  const bookexec = new Book(book.title, book.name, book.pages, book.readstatus);
+  const node = document.getElementById('title');
+  const template = `<div class="card m-4 p-relative text-center">
+                    <div class="card-body mt-5">
+                    <h5 class="card-title">${bookexec.name}</h5>
+                    <p class="card-text">${bookexec.title}</p>
+                    
+                    <p class="card-text text-center"><span class="font-weight-bolder">Number of pages:  ${bookexec.pages}</span></p>
+                    <a href="#" class="btn btn-info ptl-0">${bookexec.info()}</a>
+                    <a href="#" class="btn btn-danger remove w-100">Remove Book</a>
+                    </div>
+                </div>`;
+  render(template, node);
+});
+
+
+button.addEventListener('click', (e) => {
+  e.preventDefault();
+
+
+  if (form.book_title.value.length < 3 || form.author_name.value.length < 3) {
+    alert('Sorry Book title and Author name should be at least 3 characters long!');
```

Comment:
Instead of this, you could add the validation to your input, we have `minlength` attribute, you can [read about it here](https://html.spec.whatwg.org/multipage/input.html#the-maxlength-and-minlength-attributes). If you want custom messages like you used, check [here on how to go about the length validation](https://stackoverflow.com/questions/10281962/is-there-a-minlength-validation-attribute-in-html5).